### ISSUE TITLE
[Legitimate Site Blocked] unilayer.app

### ISSUE THREAD

**Issue by dehavenmi** [MetaMask/eth-phishing-detect#5828]
https://unilayer.app/

Please continue to flag this crypto site as it is a scam IDO and Airdrop.  Many of us have lost $1000s and were promised airdrops and never happened.  Their IDO was another lie and misrepresented and another loss of huge amount of money.  

Contact form on website doesnt work.  Owner/Developers/Admins deleted Telegram Channels and have refused to discuss any issues via Twitter with any Layer owners.  Many people are super angry and their website and offerings cant be trusted anymore!

SCAM and please never unblock them as others will get hurt!

Many Thanks!

**Comment by devenblake** [comment]
Duplicate of #5801.

**Comment by btcgeo** [comment]
This is not true, our social media channels are still LIVE, you can verify it here:

https://t.me/Unilayer
https://twitter.com/Unilayer_

**Comment by dehavenmi** [comment]
Fuck off and die scamming piece of shit!  U don't deserve to even live or breath the same air.  Scammer!
________________________________
